languages/javascript/exercises/concept/nullability/.docs/hints.md

```diff
@@ -0,0 +1,4 @@
+## 1. Create the badge text.
+
+- Be careful about the differences between equality (`==`) and identity (`===`) operators.
+- Strings in JavaScript have several methods to change between upper and lower cases.
```

Comment:
You could consider linking to a resource for the upper and/or lower case conversion.